JASP-Engine/JASP/R/summarystatscorrelationbayesianpairs.R

```diff
@@ -252,25 +253,18 @@ SummaryStatsCorrelationBayesianPairs <- function(jaspResults, dataset=NULL, opti
         options[["plotPriorPosterior"]] <- options[[currentName]]
       }
       
-      
-      if (currentName=="plotPriorAndPosteriorAdditionalInfo") {
+      if (currentName=="plotPriorAndPosteriorAdditionalInfo") 
```

Comment:
This could be done even nicer with a switch is what I wanted to say, but even better is a list:
```R
converter <- list( #I would align this all better if I was in an actual editor, but alas...
  plotPriorAndPosteriorAdditionalInfo="plotPriorPosteriorAddTestingInfo",
  plotBayesFactorRobustness="plotBfRobustness",
  plotBayesFactorRobustnessAdditionalInfo="plotBfRobustnessAddInfo",
  priorWidth="kappa")

options[[converter[[currentName]] ]] <- options[[currentName]]
```

Although I have the feeling that maybe some options ought to be renamed instead of doing this...